sympy/core/sympify.py

```diff
@@ -51,6 +51,45 @@ class CantSympify(object):
     pass
 
 
+def convert_numpy_types(a):
+    """
+    Converts a numpy datatype input to an appropriate sympy type.
+
+    Examples
+    ========
+
+    >>> import sympy as sp
+    >>> import numpy as np
```

Comment:
This means all doctests will pull in numpy.  If I understand correctly, this is to be avoided.

This function is not "public facing" so maybe it doesn't need these (otherwise nice) doctests.

(I could be wrong about this).

Reply:
I've changed this to only import the functions that are used in the doctests. (Is that even necessary? I don't think I fully understand doctests.)